ethabi/src/decoder.rs

```diff
@@ -118,7 +118,7 @@ fn decode_param(param: &ParamType, data: &[u8], offset: usize) -> Result<DecodeR
 			let bytes = take_bytes(data, offset, len)?;
 			let result = DecodeResult {
 				token: Token::FixedBytes(bytes),
-				new_offset: offset + len,
+				new_offset: offset + 32,
```

Comment:
It would be good to have a comment here why we advance by 32 rather than `len`; in particular, is `len` always going to be less than 32?

Reply:
@lutter Done!